integration-tests/actions/actions.go

```diff
@@ -54,6 +54,26 @@ func FundChainlinkNodes(
 	return client.WaitForEvents()
 }
 
+// FundChainlinkNodesAddress will fund all of the provided Chainlink nodes address at given index with a set amount of native currency
+func FundChainlinkNodesAddress(
```

Comment:
Does it make sense to replace `FundChainlinkNodes` with this method with a keyIndex of 0?